src/components/configuration/sdk-config-docs/cordova.js

```diff
@@ -101,6 +101,25 @@ const framework = docsPrefix => {
             ]
           }
         ]
+      },
+      security: {
+        serviceLogoUrl: '/img/security.svg',
+        serviceName: 'Security',
+        serviceDescription:
+          'Installs a security service which gives developers the ability to control their apps from the Mobile Security Console.',
+        setupText: 'Security SDK setup',
+        docsLink: `${docsPrefix}/security.html`,
+        steps: [
+          {
+            introduction: 'Setting up Mobile Security service SDK',
+            commands: [
+              [
```

Comment:
You are missing steps I think. I believe you should have all 3 steps from [AeroGear docs](https://docs.aerogear.org/aerogear/latest/security.html#importing-the-libraries)